src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-container.component.ts

```diff
@@ -119,6 +121,7 @@ export class CodeEditorTutorAssessmentContainerComponent implements OnInit, OnDe
         private programmingExerciseService: ProgrammingExerciseService,
     ) {
         translateService.get('artemisApp.assessment.messages.confirmCancel').subscribe((text) => (this.cancelConfirmationText = text));
+        translateService.get('artemisApp.assessment.messages.acceptComplaintWithoutMoreScore').subscribe((text) => (this.acceptComplaintWithoutMoreScoreText = text));
```

Comment:
Optional / Followup: This does change the text if the user decides to change the language while on this page. This should rarely happen, but it would be better to translate right before using the text.